README.adoc

```diff
@@ -26,27 +26,21 @@ You’ll explore how to check the health of a microservice with MicroProfile Hea
 == What you'll learn
 
 You will learn how to use the MicroProfile Health to report the health of
-a microservice. A health check allows your microservice to participate in the decision-making process about its health.
+a microservice. A health check allows your microservice to participate in the decision-making process about
+its health.
```

Comment:
You will learn how to use MicroProfile Health to report the health of a microservice. <--Don't need the word "the" here. :)

Reply:
without "the" ? hmm.. it sounds weird  but ok. Fixed